Jenkinsfile.release

```diff
@@ -31,10 +39,13 @@ ansiColor('xterm') {
           sshagent (credentials: ['0f7ec9c9-99b2-4797-9ed5-625572d5931d']) {
             // add the ssh host key for the package uploader step
             // These environment variables are defined in the "Properties Content" section for the job.
```

Comment:
What is the reason for not using `withCredentials` for these?

Reply:
Why would we use withCredentials ? :) For ssh credentials, my understanding is that using an sshagent is the way to provide ssh credentials in Jenkins